app/views/moments/show.html.erb

```diff
@@ -43,6 +43,18 @@
   </div>
 <% end %>
 
+<% if @resources.any? %>
+<div class="smallMarginTop">
+  <div class="label"><%= label_tag "What resources could help?" %> </div>
+  <ul>
+    <% @resources.take(3).each do |item|  %>
+      <li><%= link_to item['name'], item['link'] %></li>
+    <%end %>
+    <li><%= link_to  'More...', "/resources?#{@resources_tags}" %></li>
```

Comment:
"More..." should be in our translation files. Alternatively, you could use the existing `t('load_more')` key.